spec/spec_helper.rb

```diff
@@ -1 +1,23 @@
+if ENV['COVERAGE']
```

Comment:
not sure I understand. these requires are specifically needed for simplecov, why not do them in the setup-simplecov task?